python/port/mod/os/modos.cpp

```diff
@@ -111,5 +111,4 @@ mp_obj_t modos_listdir(void) {
   }
   
   return list;
-}
-
+}
```

Comment:
?

Reply:
Just a mistake, I will revert the modification in the next commit